docs/getting-started-apm-server.asciidoc

```diff
@@ -0,0 +1,285 @@
+------------------------------------------------
+curl -L -O {downloads}/apm-server-{version}-darwin-x86_64.tar.gz
+tar xzvf apm-server-{version}-darwin-x86_64.tar.gz
+------------------------------------------------
+
+endif::[]
+
+[[linux]]
+*linux:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of APM Server has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O {downloads}/apm-server-{version}-linux-x86_64.tar.gz
+tar xzvf apm-server-{version}-linux-x86_64.tar.gz
+------------------------------------------------
+endif::[]
+
+[[docker]]
+*docker:*
+
+See <<running-on-docker, Running on Docker>> for deploying Docker containers.
+
+[[installing-on-windows]]
+*win:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of APM Server has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+. Download the APM Server Windows zip file from the
+https://www.elastic.co/downloads/apm/apm-server[downloads page].
+
+. Extract the contents of the zip file into `C:\Program Files`.
+
+. Rename the `apm-server-<version>-windows` directory to `APM-Server`.
+
+. Open a PowerShell prompt as an Administrator (right-click the PowerShell icon and select *Run As Administrator*).
+If you are running Windows XP, you may need to download and install PowerShell.
+
+. From the PowerShell prompt, run the following commands to install APM Server as a Windows service:
++
+[source,shell]
+----------------------------------------------------------------------
+PS > cd 'C:\Program Files\APM-Server'
+PS C:\Program Files\APM-Server> .\install-service-apm-server.ps1
+----------------------------------------------------------------------
+
+NOTE: If script execution is disabled on your system,
+you need to set the execution policy for the current session to allow the script to run.
+For example: `PowerShell.exe -ExecutionPolicy UnRestricted -File .\install-service-apm-server.ps1`.
+
+endif::[]
+
+// *******************************************************
+// STEP 2
+// *******************************************************
+
+[[apm-server-configuration]]
+=== Step 2: Configure APM Server
+
+If you've installed APM Server yourself,
+you can edit the `apm-server.yml` configuration file to customize APM Server to your needs.
+The location of this file varies by platform -- see <<directory-layout>> for help with locating it.
+
+All available configuration options are outlined in
+{apm-server-ref-v}/configuring-howto-apm-server.html[configuring APM Server].
+
+[float]
+[[index-template-config]]
+==== Index template configuration
+
+In Elasticsearch, {ref}/indices-templates.html[index templates] are used to define settings
+and mappings that determine how fields should be analyzed.
+
+The recommended index template file for {beatname_uc} is installed by the {beatname_uc} packages.
```

Comment:
I'd said before I don't have a strong opinion about this part, but given how many issues users are facing because of this, I changed my mind. 
I think this is good. 99% of times I prefer concision, but here I'd add some points about:

- The APM UI relies on specific mappings, and changing them might break it.
- Users can run `apm-server -e setup template` to force a template overwrite; and `apm-server export template` to see what is the template bundled with apm-server and compare it with the actual template in their elasticsearch instance.
- Note that index templates are not applied retrospectively, but only to indices created from then on. For APM, this usually means one day, or whatever indicates the ILM policy (if there is one).
- Consider a troubleshooting section in Kibana mentioning that certain errors happen because of having the wrong template installed, linking back to this section and ILM manual setup. Feel free to sync up with the apm-ui team for this.

Reply:
Added in your recommendations, thanks!

> Consider a troubleshooting section in Kibana mentioning that certain errors happen because of having the wrong template installed, linking back to this section and ILM manual setup. Feel free to sync up with the apm-ui team for this.

I'm going to leave the last bullet point for when I tackle https://github.com/elastic/apm-server/issues/2470